src/script/page/MainContent/panels/preferences/AVPreferences.tsx

```diff
@@ -43,22 +48,47 @@ const AVPreferences = ({
   propertiesRepository,
   callingRepository,
 }: AVPreferencesProps) => {
+  const [isCheckingPermissions, setCheckingPermissions] = useState(false);
   const deviceSupport = useKoSubscribableChildren(devicesHandler?.deviceSupport, [
     MediaDeviceType.AUDIO_INPUT,
     MediaDeviceType.AUDIO_OUTPUT,
     MediaDeviceType.VIDEO_INPUT,
   ]);
 
+  const initializeMediaDevices = async () => {
+    setCheckingPermissions(true);
+    try {
+      await streamHandler.requestMediaSreamAccess(true).then(stream => {
+        devicesHandler?.initializeMediaDevices().then(() => {
+          stream?.getTracks().forEach(track => track.stop());
+        });
+      });
+    } catch (error) {
+      logger.warn(`Initialization of media devices failed: ${error.message}`, error);
+    } finally {
+      setCheckingPermissions(false);
+    }
+  };
+
+  useEffect(() => {
+    initializeMediaDevices();
+  }, []);
```

Comment:
I think I need some help here. I'm not an Reakt developer. 

I'm not really happy with the bahavior either.. I start a video stream here to enforce the permissions and block the UI for it

Reply:
@przemvs I built a hook like you described. Could you please check if everything is OK.

No unmount  func because no state needs to be reset. Because ``devicesHandler: MediaDevicesHandler``  is a Singleton and uses the local storage to check its state.

And in the current implementation, the permissions are always checked when open the settings.. that's ok.